WindowsServerDocs/storage/refs/refs-overview.md

```diff
@@ -76,7 +76,8 @@ Deploying ReFS on Storage Spaces Direct is recommended for virtualized workloads
 - Deploying ReFS on Storage Spaces with shared SAS enclosures is suitable for hosting archival data and storing user documents.
 
 > [!NOTE]
-> Storage Spaces supports local non-removable direct-attached via BusTypes SATA, SAS, NVME, or attached via HBA (aka RAID controller in pass-through mode).
+> Storage Spaces **supports** *local non-removable direct-attached* via BusTypes SATA, SAS, NVME, or attached via HBA (aka RAID controller in pass-through mode).
+> Please consider **limitations** on Cluster Shared Volumes in general, or CSVs used on Storage Spaces or Storage Spaces Direct (S2D). Refer chapter functionality.
```

Comment:
Just like in my other comment, the phrase "Refer chapter functionality." uses incomplete English syntax. A better English phrasing would be:
```suggestion
> Please consider **limitations** on Cluster Shared Volumes in general, or CSVs used on Storage Spaces or Storage Spaces Direct (S2D). Refer to the functionality chapter.
```